### ISSUE TITLE
Game Database fails to load after Playnite App restart

### ISSUE THREAD

**Issue by rocket1138** [JosefNemec/Playnite#3801]
### Bug Description

Playnite fails to load the database after a restart
Themes and addons work fine but games disappear from the application after doing a library update and then a restart

### Diagnostics ID

3857908e-fb3c-498d-aba3-979506eba2b5

### Screenshots

![image_2024-08-17_022433305](https://github.com/user-attachments/assets/ca89e5e9-72c5-444a-8a23-ee11d25104a4)
Library before update

![image_2024-08-17_022723247](https://github.com/user-attachments/assets/016aacad-a616-4e50-bbed-dbdb52cb27d0)
Library after update and restart

**Comment by Jeshibu** [comment]
Does this still happen if you disable SuccessStory?

**Comment by JosefNemec** [comment]
I see some errors in the logs related to library data handling, but I would guess this is indeed caused by the latest SuccessStory update since people are having similar issues after they installed it.

**Comment by rocket1138** [comment]
just tried with SS disabled still happening

**Comment by JosefNemec** [comment]
Does it happen with all plugins disabled?

You have ton of other plugins installed that could be messing breaking this, like installation status updater, given that you have installed filter enabled, it might be setting all games to uninstalled state.

**Comment by rocket1138** [comment]
Yes still happening with all addons disabled

I  restore my backup, playnite restarts games are there i update the library and restart games are gone

**Comment by JosefNemec** [comment]
Playnite never removes games on its own, so it's most likely just game states being changed for some reason and games being filtered out. Disable all filters to see if those games are really gone.

**Comment by JosefNemec** [comment]
Also please attach `library\games.db` file from install folder.

**Comment by rocket1138** [comment]
![image_2024-08-17_035408339](https://github.com/user-attachments/assets/b241d7f8-76d9-4961-a5e7-4452553f1e14)
heres a screenshot game arent there even with filters disabled

**Comment by JosefNemec** [comment]
Please attach `library\games.db` file from install folder.

**Comment by rocket1138** [comment]
[games.zip](https://github.com/user-attachments/files/16645760/games.zip)

**Comment by JosefNemec** [comment]
I'm afraid the file is corrupted beyond repair. Do you have the original backup file from before games go missing? Can you attach that one? Also during that backup restore you mentioned, did you have plugins enabled? My only guess is that some plugin might be corrupting it.

**Comment by rocket1138** [comment]
Yes i have an old backup of my database
[backup_games.zip](https://github.com/user-attachments/files/16650489/backup_games.zip)
I tried to restore it both with plugins enabled and disabled and games still keep despairing after trying to update the library and restarting the app

**Comment by JosefNemec** [comment]
Try with this file: [games.zip](https://github.com/user-attachments/files/16650719/games.zip)

There's definitely something fishy going on with your games file. It's around 34MB while it should be just around 4MB.

**Comment by rocket1138** [comment]
Doesn't seem to work Playnite does its thing with the backup but no games show 

At this point would you recommend starting from scratch with a new installation?

I really don't want to waste development time on something that was most likely my fault.

**Comment by JosefNemec** [comment]
Yeah I have no idea what's going on, db files must be messed up in some specific way. I'm out of ideas if rebuilt file I attached doesn't work.
